File: src/game/events/UpdateMaxBombsEvent.java
package game.events;

import game.Bomberman;

/**
 * Fires a game event using Strategy Pattern;
 */
public class UpdateMaxBombsEvent implements GameEvent {
    @Override
    public void invoke(Object arg) {
        Bomberman.getMatch().getCurrentLevel().onUpdateMaxBombsGameEvent((int) arg);
    }
}

File: src/game/events/game/UpdateMaxBombsEvent.kt
package game.events.game

import game.Bomberman
import game.events.models.GameEvent


class UpdateMaxBombsEvent : GameEvent {
    override fun invoke(arg: Any?) {
        Bomberman.getMatch().currentLevel.eventHandler.onUpdateMaxBombsGameEvent(arg as Int)
    }
}